src/routers/handlers/choose_your_package_accounts/package.type.radio.options.ts

```diff
@@ -0,0 +1,94 @@
+import { PackageType } from "@companieshouse/api-sdk-node/dist/services/accounts-filing/types";
+import { logger } from "../../../utils/logger";
+
+export const fees: Record<PackageType, number> = {
+    "cic": 15,
```

Comment:
Fee's should come from config, they shouldn't be hard coded, in case they increase in the future